CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.70.0](https://github.com/aws/aws-cdk/compare/v1.69.0...v1.70.0) (2020-10-23)
+
+
+### ⚠ BREAKING CHANGES TO EXPERIMENTAL FEATURES
+
+* **cognito:** the `UserPoolIdentityProviderBase` abstract class has been removed. Use the `UserPoolIdentityProvider*` classes directly.
+
+### Features
+
+* **apigatewayv2:** vpc link and private integrations ([#10531](https://github.com/aws/aws-cdk/issues/10531)) ([0537598](https://github.com/aws/aws-cdk/commit/0537598a065c74c713d1ec58c96f09f0d66b12ca)), closes [/docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-integration.html#cfn-apigatewayv2](https://github.com/aws//docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-integration.html/issues/cfn-apigatewayv2) [#10119](https://github.com/aws/aws-cdk/issues/10119)
```

Comment:
This was reverted, right?

Reply:
Also, the entry is messed up 🙂

Reply:
err - thought i just removed this one. yes it needs to be removed. good catch!!

Reply:
done!